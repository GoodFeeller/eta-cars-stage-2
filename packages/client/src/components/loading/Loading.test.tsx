import { render, screen } from '@testing-library/react'
import Loading from './Loading'
test('Loading snapshot test', () => {
  render(<Loading />)
  const load = screen.queryByRole('div')
  expect(load).toMatchSnapshot()
})
