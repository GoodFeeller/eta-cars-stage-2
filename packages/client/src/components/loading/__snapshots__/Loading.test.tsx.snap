// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading snapshot test 1`] = `null`;
